notebooks/tutorial/02_performance.ipynb

```diff
@@ -0,0 +1,478 @@
+{
```

Comment:
Why is latok lagging in performance compared to pc-regex and gensim?

Seems like it should outperform both???

Reply:
Yes, this does warrant further investigation, including finding any current bottlenecks and addressing them.